mappings/GLImportProcessor.mapping

```diff
@@ -0,0 +1,33 @@
+CLASS net/minecraft/class_5913 GLImportProcessor
```

Comment:
Meh, this class has no package and  broke mappingpoet. I didn't expect that bad javadoc gen library to have this cool functionality!

Reply:
This has been an issue in mappingpoet for a while. We need another source of validation for classes not in a package.

Reply:
Oh, whoops! I forgot that I had to give it a package too. One moment.